momentum/changelog/4/4-2-40.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Momentum 4.2.40 Changelog"
+description: "Momentum 4.2.40 was released on 2018-??-??. This section will list all of the major changes that happened with the release of Momentum 4.2.40. Depending on installation type, all changes may not be applicable."
+---
+
+**Momentum 4.2.40** was released on 2018-??-??. This section will list all of the major changes that happened with the release of **Momentum 4.2.40**. Depending on installation type, all changes may not be applicable.
```

Comment:
I know it was like this before, but do we want to take this opportunity to remove the question marks, and just make it something like "in 2018"?

Reply:
Look at the date on the tag in the Core repo for guidance

Reply:
I just went with "in 2018". I looked at hg log for the tag 4.2.40.1 and I think the build/packaging merge may have messed the dates up? Because I see 
``` 
user:        John Peacock <john.peacock@messagesystems.com>
date:        Tue Jan 21 11:08:40 2020 -0500
```

Reply:
That was the migration date for the single repo. Try looking at the release branch was instead

> changeset:   78939:73ce27d08e16
> branch:      release/4.2.40.1
> parent:      78937:e04f819fc438
> user:        Samuel Nixon <samuel.nixon@messagesystems.com>
> date:        Tue Oct 23 20:38:08 2018 +0000
> summary:     flow: Closed <release> '4.2.40.1'.
>